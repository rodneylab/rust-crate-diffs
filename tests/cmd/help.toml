bin.name = "rust-crate-diffs"
args = "--help"
status.code = 0
