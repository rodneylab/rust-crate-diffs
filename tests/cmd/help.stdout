Check Cargo.toml dependency changes against git repo head

Usage: rust-crate-diffs [OPTIONS] <REPO_PATH>

Arguments:
  <REPO_PATH>  repo path

Options:
  -v, --verbose...  Increase logging verbosity
  -q, --quiet...    Decrease logging verbosity
  -h, --help        Print help
  -V, --version     Print version
